Android/app/src/main/java/io/realm/scanner/MainActivity.java

```diff
@@ -115,9 +115,9 @@ public void onClick(View view) {
         checkPermissionAndCopyTestAssetImage();
 
         final SyncCredentials syncCredentials = SyncCredentials.usernamePassword(ID, PASSWORD, false);
-        SyncUser.loginAsync(syncCredentials, AUTH_URL, new SyncUser.Callback() {
+        SyncUser.loginAsync(syncCredentials, AUTH_URL, new SyncUser.Callback<SyncUser>() {
             @Override
-            public void onSuccess(SyncUser user) {
+            public void onSuccess(@NonNull SyncUser user) {
```

Comment:
why adding the annotation I thought by default it is not null.

Reply:
You get warnings if you don't add it.  It's just a factor of implementing the interface on the Scanner app side.   I think AS sees that I'm implementing a method with a signature that promises the args won't be null, but my override (implementation) doesn't also specify that in it's signature.

Totally not an issue here but AS still complains and I don't like the highlights or suppressing the warning, and Scanner doesn't have NonNullByDefault setup, so I just added the @NonNull.  

Long explanation to say, I see your point, not absolutely necessary, but that's why I added it. :-)